cmd/connect.go

```diff
@@ -38,17 +38,7 @@ peers & swapping data.`,
 		},
 	}
 
-	cmd.Flags().IntVarP(&o.DisconnectAfter, "disconnect-after", "", 0, "duration to keep connected in seconds, 0 means run indefinitely")
```

Comment:
yep yep yep. I'm glad this is happening. It'll greatly reduce the complexity around the source of truth when it comes to configuration b/c we no longer need to think about "ephemeral config modification"